CSS/Styles.css

```diff
@@ -109,4 +110,251 @@ body {
+    grid-column-start: 1;
+    grid-column-end: 2;
+    grid-row-start: 7;
+    grid-row-end: 8;
+  }
+
+  .mobile-portfolio > div > div {
+    background-color: #1a2236;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .mobile-portfolio > div > div > h2 {
+    margin: 0;
+    font-weight: 300;
+    font-size: 20px;
+    line-height: 80%;
+    color: #fff;
+    margin-top: 20px;
+    margin-left: 20px;
+  }
+
+  .mobile-portfolio > div > div > ul {
+    padding: 0;
+  }
+
+  .mobile-portfolio > div > div > ul > li {
+    display: inline;
+    text-decoration: none;
+    font-weight: 300;
+    font-size: 10.33px;
+    line-height: 80%;
+    color: #1a2236;
+    background-color: #98dede;
+    height: 20px;
+  }
+
+  .li-htm {
+    width: 69px;
+    margin-left: 20px;
+  }
+
+  .li-rub {
+    width: 87px;
+    margin-left: 10px;
+  }
+
+  .li-jav {
+    width: 74px;
+    margin-left: 10px;
+  }
+
+  .mobile-portfolio > div > div > button {
+    width: 178px;
+    margin-left: 20px;
+    margin-bottom: 20px;
+    height: 44px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid #fff;
+    background-color: #1a2236;
+  }
+
+  .mobile-portfolio > div > div > button > span {
+    font-family: 'Poppins', sans-serif;
+    color: #fff;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .mobile-portfolio > div > div > button:hover {
+    border: 1px solid #1a2236;
+    background-color: #fff;
+  }
+
+  .mobile-portfolio > div > div > button:hover > span {
+    color: #1a2236;
+  }
+
+  .mobile-portfolio > div > div > button:active {
+    border: 2px solid #1a2236;
+  }
+
+  /* About Section */
+  .mobile-about {
+    display: flex;
+    flex-direction: column;
+    margin-top: 56px;
+    margin-left: 16px;
+  }
+
+  .mobile-about > div > h1 {
+    font-weight: 400;
+    font-size: 50px;
+    color: #1a2236;
+    margin: 0;
+    line-height: 100%;
+  }
+
+  .mobile-about > div > p {
+    margin-top: 20px;
+    font-weight: 400;
+    font-size: 14px;
+    color: #1a2236;
+    line-height: 21px;
+    margin-bottom: 32px;
+  }
+
+  .mobile-about > div > button {
+    width: 288px;
+    height: 50px;
+    background-color: #1a2236;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-weight: 500;
+    font-size: 16px;
+    color: #fff;
+    line-height: 24px;
+  }
+
+  .mobile-about > div > button > span {
+    margin-left: 20px;
+  }
+
+  .mobile-about > div > button:hover {
+    border: 1px solid #1a2236;
+    background-color: #fff;
+    color: #1a2236;
+  }
+
+  .about-entry {
+    width: 289px;
+    height: 289px;
+    color: #fff;
+    margin-top: 16px;
+  }
+
+  .languages {
+    background-color: #1c528a;
+    margin-top: 40px;
+  }
+
+  .frameworks {
+    background-color: #e2505c;
+  }
+
+  .skills {
+    background-color: #febf52;
+  }
+
+  .about-entry > h3 {
+    margin-top: 50px;
+    margin-left: 50px;
+    margin-bottom: 0;
+  }
```

Comment:
The text in this entire section doesn't have the same weight as in the template design. Please fix it.

<img width="557" alt="Screen Shot 2022-06-29 at 18 08 23" src="https://user-images.githubusercontent.com/82426682/176560478-314ceae8-9448-413a-8fcf-eb2ae7d895f0.png">